client/my-sites/people/people-section-nav/index.jsx

```diff
@@ -111,14 +111,19 @@ class PeopleSectionNav extends Component {
 				path: '/people/viewers/' + siteFilter,
 				id: 'viewers',
 			},
+			{
+				title: translate( 'Invites', { context: 'Filter label for people list' } ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **59** times:
`translate( 'Invites' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?

Reply:
I guess strings with separate context need to be translated separately?  I think it's fine to remove the context here.

Reply:
Yeah, I struggled with that too, and opted to follow the pattern of the nearby translations... but this and the other translations should really be comments, not context. I'll just remove it because even comments aren't really necessary in this case either.